redfish-core/lib/hypervisor_system.hpp

```diff
@@ -938,32 +938,13 @@ inline void handleHypervisorIPv6StaticPatch(
             return;
         }
 
-        bool errorInEntry = false;
-        if (address)
-        {
-            if (!ip_util::ipv4VerifyIpAndGetBitcount(*address))
-            {
-                messages::propertyValueFormatError(asyncResp->res, *address,
-                                                   pathString + "/Address");
-                errorInEntry = true;
-            }
-        }
-        else
+        if (!address)
```

Comment:
You have to patch with an address? 
Can't just do just the gateway, etc?

Reply:
This is downstream only code which was there in 1060 as well.
this was missed as part of rebase PRs.

Reply:
>this was missed as part of rebase PRs.

+1